README.md

```diff
@@ -6,10 +6,28 @@ Soapbox provides managed web application hosting services, encapsulating best-pr
 
 ### Requirements
 
- - Go 1.8 or greater
- - Ruby 2.2 or greater
- - PostgreSQL 9.5 or greater
- - Terraform 0.9.11 or greater
+ - Go 1.8 or greater - see directions below
+ - Ruby 2.2 or greater - Install using [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/rvm/install)
+ - PostgreSQL 9.5 or greater - [Ubuntu](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-16-04) | [Mac](https://solidfoundationwebdev.com/blog/posts/how-to-install-postgresql-using-brew-on-osx)
+ - Terraform 0.9.11 or greater - Download and install Terraform from [here](https://www.terraform.io/downloads.html).
+
+### Installing Go
+
+###### Linux
+
+``` shell
+$ bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+$ gvm install go1.4
+$ gvm use go1.4
+$ gvm install go1.8.3
+$ gvm use go1.8.3
+```
+
+###### Mac OS
+
+1. Download the package file (here)[https://golang.org/dl/]
+2. Open the package file and follow the prompts.
+3. Restart any terminals you have open.
```

Comment:
Again, there is already documentation elsewhere on how to install Go -- I would rather point people to that than try to replicate it here.